packages/arb-token-bridge-ui/src/generateOpenGraphImages.tsx

```diff
@@ -0,0 +1,342 @@
+import React from 'react'
+import satori, { Font } from 'satori'
+import sharp from 'sharp'
+import fs from 'fs'
+import { ChainId, isNetwork } from './util/networks'
+import { getBridgeUiConfigForChain } from './util/bridgeUiConfig'
+import { orbitMainnets, orbitTestnets } from './util/orbitChainsList'
+
+const dimensions = {
+  width: 1200,
+  height: 627
+} as const
+
+async function getFonts(): Promise<Font[]> {
+  const unica77_regular = fs.readFileSync('./src/font/Unica77LL-Regular.otf')
+  const unica77_medium = fs.readFileSync('./src/font/Unica77LL-Medium.otf')
+
+  return [
+    {
+      name: 'Unica77',
+      data: unica77_regular,
+      weight: 400,
+      style: 'normal'
+    },
+    {
+      name: 'Unica77',
+      data: unica77_medium,
+      weight: 500,
+      style: 'normal'
+    }
+  ]
+}
+
+type Chain = ChainId | number
+
+type ChainCombination = [Chain, Chain]
+
+const chainToLogo: { [key: Chain]: string } = {
+  [ChainId.Ethereum]: 'https://l2beat.com/icons/ethereum.png',
+  [ChainId.ArbitrumOne]: 'https://l2beat.com/icons/arbitrum.png',
+  [ChainId.ArbitrumNova]: 'https://l2beat.com/icons/nova.png'
+}
+
+const configs: ChainCombination[] = [
+  [ChainId.Ethereum, ChainId.ArbitrumOne],
+  [ChainId.Ethereum, ChainId.ArbitrumNova],
+  [ChainId.ArbitrumOne, 660279],
```

Comment:
Why do we mention XAI separately since we already have orbit mainnets considered in the code that follows?

Reply:
missed deleting it, great catch, thanks!